forgerock-auth/src/androidTest/java/org/forgerock/android/auth/callback/DeviceBindingListAndUnbind.java

```diff
@@ -166,14 +160,17 @@ public void onException(Exception e) {
 
     @Test
     public void testListAndDeleteKeys() throws ExecutionException, InterruptedException, IOException, ApiException {
+        bindDevice("bind-pin"); // Bind the device with APPLICATION_PIN
+        bindDevice("bind");     // Bind the device with NONE
```

Comment:
Now binding App Pin Key and Keystore key will remove the previous key, so only one key will persist on the client side. I have changed the test to test only one key per person per device.